tools/testbench/include/testbench/file.h

```diff
@@ -52,5 +52,5 @@ struct sof_ipc_comp_file {
 	struct sof_ipc_comp_config config;
 	char *fn;
```

Comment:
@singalsu it is not clear *why* this fix is needed. What is the build error message?

Reply:
I wonder if other distros/distro versions give this. My gcc is version 9.2.1 and the error is:

```
/usr/bin/cc  -DRELATIVE_FILE=\"panic.c\" -I/home/singalsu/work/current/sof/sof/tools/testbench/include -I/home/singalsu/work/current/sof/sof/tools/testbench/build_testbench/sof_ep/install/include -I/home/singalsu/work/current/sof/sof/tools/testbench/build_testbench/sof_parser/install/include  -g -O3 -Wall -Werror -Wl,-EL -Wmissing-prototypes -Wimplicit-fallthrough=3 -o CMakeFiles/testbench.dir/panic.c.o   -c /home/singalsu/work/current/sof/sof/tools/testbench/panic.c
/home/singalsu/work/current/sof/sof/tools/testbench/file.c: In function ‘file_new’:
/home/singalsu/work/current/sof/sof/tools/testbench/file.c:408:11: error: converting a packed ‘struct sof_ipc_comp’ pointer (alignment 1) to a ‘struct sof_ipc_comp_file’ pointer (alignment 8) may result in an unaligned pointer value [-Werror=address-of-packed-member]
  408 |   (struct sof_ipc_comp_file *)comp;
      |           ^~~~~~~~~~~~~~~~~
In file included from /home/singalsu/work/current/sof/sof/tools/testbench/build_testbench/sof_ep/install/include/sof/audio/pipeline.h:13,
                 from /home/singalsu/work/current/sof/sof/tools/testbench/build_testbench/sof_ep/install/include/sof/audio/buffer.h:12,
                 from /home/singalsu/work/current/sof/sof/tools/testbench/build_testbench/sof_ep/install/include/sof/audio/component.h:19,
                 from /home/singalsu/work/current/sof/sof/tools/testbench/file.c:17:
/home/singalsu/work/current/sof/sof/tools/testbench/build_testbench/sof_ep/install/include/ipc/topology.h:59:8: note: defined here
   59 | struct sof_ipc_comp {
      |        ^~~~~~~~~~~~
In file included from /home/singalsu/work/current/sof/sof/tools/testbench/file.c:22:
/home/singalsu/work/current/sof/sof/tools/testbench/include/testbench/file.h:50:8: note: defined here
   50 | struct sof_ipc_comp_file {
      |        ^~~~~~~~~~~~~~~~~

```

Reply:
I use for testbench build command "scripts/host-build-all.sh" from SOF top level.

Reply:
@singalsu No error on my machine, using default compiler from Ubuntu 18.04 (7.4.0). Anyhow, the error looks legit please mention in the commit message that you are getting this error with gcc 9.2.1 and the error message is
```
sof/tools/testbench/file.c:408:11: error: converting a packed ‘struct sof_ipc_comp’ pointer (alignment 1) to a ‘struct sof_ipc_comp_file’ pointer (alignment 8) may result in an unaligned pointer value [-Werror=address-of-packed-member]
  408 |   (struct sof_ipc_comp_file *)comp;

```

Other than that it looks good to me.

Reply:
Yep, good idea. I'll add that!

Reply:
Done.